src/vs/workbench/services/host/browser/browserHostService.ts

```diff
@@ -327,6 +327,31 @@ export class BrowserHostService extends Disposable implements IHostService {
 		}
 	}
 
+	async exitFullScreen(): Promise<void> {
+
+		// Chromium
+		if (document.fullscreen !== undefined) {
```

Comment:
Looking at the above method `toggleFullScreen` I feel that code can be shared between the two methods, since `toggleFullScreen` also exits fullscreen if already fullscreen.